add ros2doctor README (#318)
ros2/ros2cli: ros2doctor/README.md

# ros2doctor

> This folder contains the source code of ros2doctor, part of the ROS 2 command line interface tools included with a standard install of any ROS 2 distro. ros2doctor command line tool is similar to roswtf in ROS system. It will examine your ROS 2 setup, such as distribution and platform, environment variables, network etc. and look for potential issues.

Reviewer: ```suggestion
This folder contains the source code of ros2doctor, part of the ROS 2 command line interface tools included with a standard install of any ROS 2 distro. 
ros2doctor command line tool is similar to roswtf in ROS system. 
It will examine your ROS 2 setup, such as distribution and platform, environment variables, network etc. and look for potential issues.
```
Each sentence has to be on a new line

# ros2doctor

> This folder contains the source code of ros2doctor.

Reviewer: ```suggestion
This folder contains the source code for ros2doctor.
```

> It is part of the ROS 2 command line interface tools included with a standard install of any ROS 2 distro.

Reviewer: ```suggestion
It is one of the ROS 2 command line interface tools included with a standard install of any ROS 2 distro.
```

> ros2doctor command line tool is similar to roswtf in ROS system.

Reviewer: ```suggestion
`ros2doctor` is similar to `roswtf` from ROS 1.
```
I'm not sure if this is true or not, but do you mean it's similar to a tool from ROS 1? If that's not right then ignore this!

> It will examine your ROS 2 setup, such as distribution and platform, network interface etc. and look for potential issues in a running ROS system.

Reviewer: ```suggestion
It will examine your ROS 2 setup, such as distribution, platform, network interface, etc., and look for potential issues in a running ROS 2 system.
```
